src/DeconvUtils.jl

```diff
@@ -90,7 +90,7 @@ function approxDeconv(fcto::DFGFactor,
     #
     
     # lambda with which to find best measurement values
-    hypoObj = (res, tgt) -> (targeti_.=tgt; onehypo!(res) )
+    hypoObj = (tgt) -> (targeti_.=tgt; onehypo!() )
```

Comment:
i missed this before, same deal: we need standardized lambda interface `(res,x)->obj...`

Reply:
ignore